src/bootstrap.jl

```diff
@@ -165,10 +165,12 @@ function allpars(bsamp::MixedModelBootstrap{T}) where {T}
 end
 
 function Base.getproperty(bsamp::MixedModelBootstrap, s::Symbol)
-    if s ∈ [:objective, :σ, :θ]
+    if s ∈ [:objective, :σ, :θ, :se]
         getproperty.(getfield(bsamp, :bstr), s)
     elseif s == :β
         tidyβ(bsamp)
+    elseif s == :coeftable
+        tidyβsezp(bsamp)
```

Comment:
this doesn't line up with the current function name `coeftable` below

Reply:
You're right - I changed the name then forgot to search for all occurrences of the old name.  Will fix.